### ISSUE TITLE
preset-env for CSS

### ISSUE THREAD

**Issue by petamoriken** [swc-project/swc#4495]
### Describe the feature

Like Babel's preset-env, PostCSS also has preset-env. By adding that feature to swc, CSS, like JS, can anticipate new features based on the specification.

### Babel plugin or link to the feature description

https://preset-env.cssdb.org/

**Comment by alexander-akait** [comment]
Yes, it is in roadmap, currently focused on autoprefixer - mostly ported except removing outdated prefixes, but I am working on it

**Comment by kdy1** [comment]
Closing as not planned.
